src/Dependencies/Collections/Internal/HashHelpers.cs

```diff
@@ -94,7 +94,7 @@ public static int ExpandPrime(int oldSize)
             ulong.MaxValue / divisor + 1;
 
         /// <summary>Performs a mod operation using the multiplier pre-computed with <see cref="GetFastModMultiplier"/>.</summary>
-        /// <remarks>This should only be used on 64-bit.</remarks>
+        /// <remarks>PERF: This should only be used on 64-bit.</remarks>
```

Comment:
Perhaps reword to be "PERF: This is fastest on 64-bit"? Right now it implies that we're just using it wrong.

Reply:
Yep, I expanded the previous note to this:

> PERF: This improves performance in 64-bit scenarios at the expense of performance in 32-bit scenarios. Since we only build a single AnyCPU binary, we opt for improved performance in the 64-bit scenario.